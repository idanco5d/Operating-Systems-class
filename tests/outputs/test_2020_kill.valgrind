==554== Memcheck, a memory error detector
==554== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==554== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==554== Command: /mnt/c/Users/illus/os/hw/1/smash
==554== Parent PID: 523
==554== 
==554== 
==554== FILE DESCRIPTORS: 4 open (3 std) at exit.
==554== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_2020_kill.valgrind
==554==    <inherited from parent>
==554== 
==554== 
==554== HEAP SUMMARY:
==554==     in use at exit: 0 bytes in 0 blocks
==554==   total heap usage: 439 allocs, 439 frees, 107,734 bytes allocated
==554== 
==554== All heap blocks were freed -- no leaks are possible
==554== 
==554== For lists of detected and suppressed errors, rerun with: -s
==554== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
