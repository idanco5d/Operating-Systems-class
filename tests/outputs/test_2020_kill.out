smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> signal number 9 was sent to pid 8871
smash> smash> smash> smash> smash> [1] sleep 7& : 9097 1 secs
smash> smash> [1] sleep 7& : 9097 1 secs
[2] sleep 5& : 9150 0 secs
smash> smash> [1] sleep 7& : 9097 2 secs
[2] sleep 5& : 9150 1 secs
[3] sleep 3& : 9151 1 secs
smash> smash> smash> smash> smash> [1] sleep 10 & : 9817 3 secs
smash> signal number 9 was sent to pid 9817
smash> smash> smash> smash> smash: got ctrl-Z
smash: process 10116 was stopped
smash> [1] sleep 5& : 10034 4 secs
[2]           sleep        10   & : 10077 3 secs
[3] sleep 15 : 10116 0 secs (stopped)
smash> signal number 9 was sent to pid 10034
smash> [2]           sleep        10   & : 10077 4 secs
[3] sleep 15 : 10116 1 secs (stopped)
smash> signal number 9 was sent to pid 10077
smash> [3] sleep 15 : 10116 2 secs (stopped)
smash> signal number 9 was sent to pid 10116
smash> smash> smash> smash: got ctrl-C
smash: process 10365 was killed
smash> signal number 9 was sent to pid 10364
smash> smash: sending SIGKILL signal to 0 jobs:
